com.remarkable.reMarkable.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.remarkable.reMarkable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "reMarkable",
+    "finish-args": [
+        "--filesystem=home",
+        "--socket=x11",
```

Comment:
you'll also want --share=ipc here for X11 apps.

Reply:
Fixed